.vscode/launch.json

```diff
@@ -8,22 +8,15 @@
             "name": "Launch Tests",
             "type": "go",
             "request": "launch",
-            "program": "${fileDirname}",
-            "mode": "test",
+            "program": "${fileDirname}/__debug_bin",
+            "mode": "exec",
             "env": {},
             "args": ["-test.v"],
-            "showLog": true
-        },
-        {
-            "name": "Launch Tests with race check",
-            "type": "go",
-            "request": "launch",
-            "program": "${fileDirname}",
-            "mode": "test",
-            "env": {},
-            "buildFlags": "-race",
-            "args": ["-test.v"],
-            "showLog": true
+            "showLog": true,
+            // https://github.com/golang/vscode-go/blob/master/docs/debugging.md#debug-a-package-test-as-root
```

Comment:
Question please: 💡 What is the relevance of this `.vscode/launch` file with this PR? aka if end-users editor is something else - say - `vim` or anything else do we need some kind of small doc for fyi?